spec/models/post.spec.js

```diff
@@ -38,6 +38,30 @@ describe('Post model', function() {
     });
   });
 
+
+  it('can add a like', function(done) {
+    const post = new Post({message: 'some message'});
+
+    post.save(function(err) {
+      expect(err).toBeNull();
+
+      Post.updateOne(
+          {'message': 'some message'},
+          {$set: {'likes': '1'}},
+          function(err) {
+            expect(err).toBeNull();
+
+        Post.find(function(err, posts) {
+          expect(err).toBeNull();
+
+          expect(posts[0]).toMatchObject({
+            message: 'some message'});
+          done();
```

Comment:
Is the expect here checking the likes on the post?

Reply:
it checks that first object of posts collection matches the object, which includes the number of likes in the document